sdk-lib/src/main/java/cash/z/ecc/android/sdk/db/entity/Account.kt

```diff
@@ -11,11 +11,11 @@ data class Account(
 
     val account: Int? = 0,
 
-    @ColumnInfo(name = "extfvk")
-    val extendedFullViewingKey: String = "",
+    @ColumnInfo(name = "ufvk")
+    val unifiedFullViewingKey: String? = "",
 
-    val address: String = "",
+    val address: String? = "",
 
     @ColumnInfo(name = "transparent_address")
-    val transparentAddress: String = ""
+    val transparentAddress: String? = ""
```

Comment:
Do we still need this if we have the UFVK?

Reply:
No, but that requires a separate database change (specifically https://github.com/zcash/librustzcash/issues/489).